plugins/syntaxtest_dev.py

```diff
@@ -428,6 +428,12 @@ class AssignSyntaxTestSyntaxListener(sublime_plugin.EventListener):
     PLAIN_TEXT = "Packages/Text/Plain text.tmLanguage"
 
     def on_load(self, view):
+        if view.size() == 0 and view.file_name().startswith(sublime.packages_path() + '/'):
+            sublime.set_timeout(lambda: self.assign_syntax(view), 100)
```

Comment:
Hard timeouts like this are usually a thing we should try to avoid. Could we instead loop the function to itself and check for `view.is_loading()` instead of `view.size() == 0`? Unfortunately I wasn't able to reliably reproduce the issue to check whether that also works myself.

```suggestion
        if view.is_loading() and view.file_name().startswith(sublime.packages_path() + '/'):
            sublime.set_timeout(lambda: self.on_load(view), 100)
```

Reply:
Was able to reproduce. Unfortuantely, `view.is_loading` is `False` despite the view still being empty, so my idea doesn't work here.
I honestly wish `View.is_loading` would be useful *anywhere*, but so far all I can remember is getting disappointed by it.